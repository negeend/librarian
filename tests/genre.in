GENRE Fantasy
genre Fantasy
ADD COLLECTION books.csv
GENRE Fantasy
GENRE FANTASY                       
genre Historical Fiction
GENRE Romance
Exit