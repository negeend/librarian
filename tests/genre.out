user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: To Kill a Mockingbird (Harper Lee)

user: No books with genre Romance.

user: Ending Library process.
